### ISSUE TITLE
forwardGeocode option 'countries' inconsistent with mapbox docs 'country'

### ISSUE THREAD

**Issue by davidfurlong** [mapbox/mapbox-sdk-js#326]
Seems rather unnecessary and confusing if intentional

https://docs.mapbox.com/api/search/#forward-geocoding

https://github.com/mapbox/mapbox-sdk-js/blob/c0ae1604f9f05a91335c70a8535597e1517c1acb/docs/services.md#forwardgeocode

**Comment by andrewharvey** [comment]
I agree it's confusing, but I think what we have here in the SDK using `countries` and accepting an array is better, so I'd rather we keep using `countries` here.

Perhaps the API could be updated to accept countries, with country still accepted for compatibility.
